lucien-ios/APIClient/LucienAPIClient.swift

```diff
@@ -162,4 +162,23 @@ final class LucienAPIClient: APIClient {
             }
         }
     }
+
+    /**
+     Sends a request to the Lucien server to edit a comic book.
+     - parameter comic: Comic Object
+     - parameter completion:
+     */
+    func editComicBook(comicID: String, comic: Comic, completion: @escaping (Result<Error?>) -> Void) {
```

Comment:
Using the type `Result<Error?>` seems redundant. Result.failure captures an error already and if it's a success, there would be no Error...